README.md

```diff
@@ -67,6 +67,8 @@ $ dune exec ./sha1sum.exe -- sha1sum.ml
 fe6e6639a817c23857b507e2d833ec776f23f327
 ```
 
+A further example is given in the examples directory.
```

Comment:
The example above is like what you propose. It will be nice to show up something more complex like: digest something in parallel (OCaml 4 & OCaml 5) with `digest_bigstring`. Digestif has a nice property to release the GC when it's about digest a bigstring.